cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillPreparer.java

```diff
@@ -368,6 +370,12 @@ public SecureTwillPreparer withSecretDisk(String runnableName, SecretDisk... sec
     return this;
   }
 
+  @Override
+  public SecureTwillPreparer withoutWorkloadIdentityEnvVar(String runnableName) {
```

Comment:
nit: Maybe we should rename this to explain what it actually does, e.g. `withNamespacedWorkloadIdentity`. We should also add a comment explaining that this feature removes the GOOGLE_APPLICATION_CREDENTIALS environment variable to enable namespaced service accounts.

Reply:
ack